### DISCUSSION TITLE
🚀 Exciting News: Amazon SageMaker Launches sagemaker-core SDK! 🚀

### DISCUSSION THREAD

**Discussion by erickb336** [aws/amazon-sagemaker-examples#4742] [Announcements]
We're thrilled to announce the release of sagemaker-core, a new Python SDK that improves how developers interact with Amazon SageMaker resources.
 
### 🔑 Key features :
• Object-oriented interface for SageMaker resources
• Resource chaining for simplified workflows
• Full parity with SageMaker APIs
• Auto code completion in popular IDEs with comprehensive documentation and type hints
 
### 🎯 Who Benefits?
Ideal for ML practitioners who need full customization of AWS primitives for their workloads, sagemaker-core reduces cognitive load and boosts productivity by handling low-level details automatically.
 
### Refer to the following links and get started today!
- [What's New Announcement](https://aws.amazon.com/about-aws/whats-new/2024/09/sagemaker-core-object-oriented-sdk-amazon-sagemaker/)
- [Example Notebooks](https://github.com/aws/amazon-sagemaker-examples/tree/default/%20%20%20%20%20%20%20%20%20sagemaker-core)
- [Technical Documentation](https://sagemaker-core.readthedocs.io)
- https://pypi.org/project/sagemaker-core/

#### Team Credits:
@mohanasudhan @mufaddal-rohawala @nargokul @benieric @pintaoz-aws @liujiaorr @marckarp 

#### Leadership Credits:
@C24IO @ShwetaSingh801 @sumedha-swamy @giuseppeporcelli

**Comment by tadeodonegana** [comment]
Great news! Thanks team 🙌🏻

**Reply by RichardScottOZ** [reply]
improvises? :)
